src/lib/swapper/swappers/LifiSwapper/getTradeQuote/getTradeQuote.ts

```diff
@@ -236,5 +236,14 @@ export async function getTradeQuote(
     }),
   )
 
+  if (promises.every(isRejected)) {
+    return Err(
+      makeSwapErrorRight({
+        message: '[LiFi: tradeQuote] - failed to get fee data',
```

Comment:
```suggestion
        message: '[LiFi: tradeQuote] - failed to get routes data',
```

Reply:
this one's deliberately about fees, the only throw inside that block is failure to get network fees. Tried to separate it out to keep the network fee catching less obscure but the refactor was larger than worth it